src/directives/NgGrid.ts

```diff
@@ -1059,6 +1072,9 @@ export class NgGrid implements OnInit, DoCheck, OnDestroy {
 	}
 
 	private _addToGrid(item: NgGridItem): void {
+		if( (this.isDragging || this.isResizing) && this.allowOverlap) {

```

Comment:
This effectively never engages the item grid when `allowOverlap` is enabled or the user is dragging or resizing an item. Is that something we want? Have you confirmed that this doesn't break anything?